ros-examples/src/aero_offboard_velocity/src/aero_offboard_velocity_node.cpp

```diff
@@ -0,0 +1,355 @@
+/**
+ * @file aero_offboard_velocity_node.cpp
+ * @brief MAVROS Offboard Control example node, written with mavros version 0.14.2, px4 flight
+ * stack and tested in Gazebo SITL & jMAVSIM.
+ */
+
+#include <ros/ros.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <mavros_msgs/CommandBool.h>
+#include <mavros_msgs/SetMode.h>
+#include <mavros_msgs/State.h>
+#include <mavros_msgs/PositionTarget.h>
+#include <unistd.h>
+#include <mavros_msgs/CommandTOL.h>
+#include <time.h>
+#include <boost/thread/thread.hpp>
+#include <ros/duration.h>
+
+void state_cb(const mavros_msgs::StateConstPtr& msg, mavros_msgs::State* current_state)
```

Comment:
Callback/Method/Function names should be verbs that indicate what it does. For eg: You can name as `save_current_state_cb()`.